srcpkgs/geonkick/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'geonkick'
+pkgname=geonkick
+version=2.5.0
+revision=1
+build_style=cmake
+hostmakedepends="pkg-config tar xz redkite"
+makedepends="redkite-devel jack-devel libsndfile-devel rapidjson
+ lv2 libX11-devel cairo-devel"
+depends="lv2"
```

Comment:
Does lv2 have to be in depends? I built and ran it w/o lv2 being in depends

Reply:
Right, removed. Thx!